docs/developer-docs/gas-cost.md

```diff
@@ -109,47 +111,45 @@ The amount of reserved cycles depends on how many bytes are allocated and on the
+If you intend to deploy canisters on high-replication subnets, your canister should be prepared for an increase in cycles prices. It is recommended to attach more cycles to a call than the current price for a high-replication subnet.
 
 ## Who is responsible for paying cycles?
 
-Canisters are responsible for paying cycles for their own canister creation, compute resources, storage resources, and execution resources. For certain canister calls, however, the canister responsible for paying the cycles may vary.
+Canisters are responsible for paying cycles for their own canister creation, compute resources, storage resources, and execution resources. For certain canister calls, the canister responsible for paying the cycles may vary.
 
-For inter-canister calls and the associated bytes, the canister sending this call is responsible for paying the required cycles. If there is an inter-canister call chain, for example, canister A → canister B → canister C, then canister A pays for the sending request-response pair between canister A and canister B, and canister B pays for sending the request-response pair between canister B and canister C. Additionally, each canister pays for processing that happens locally, meaning canister A pays for processing the response from canister B while canister B pays for processing the request from canister A.
+A canister sending an inter-canister call is responsible for paying the required cycles. In inter-canister call chains, each canister is responsible for the call that it sends. Each canister pays for processing that happens locally. 
 
-In contrast, for ingress messages, the receiving canister is responsible for paying for the consumed cycles.
+For ingress messages, the receiving canister is responsible for paying for the consumed cycles.
 
 Canisters that are created from another canister (child canisters) are responsible for paying for themselves.
 
 ## Cycles price breakdown
 
-The chart below details the cost of compute and storage transmissions as well as management canister calls. A thorough example of how the cost of running a canister on a 13-node app subnet is computed can be found [here](https://wiki.internetcomputer.org/wiki/Comparing_Canister_Cycles_vs_Performance_Counter).
-
-The USD cost for canister transmissions below is based on the cycle costs indicated for 13-node and 34-node subnets. 1 XDR is equal to 1 trillion cycles. The cost is shown in USD (XDR/USD exchange rate as of December 18, 2023). The exchange rate for USD/XDR may vary, and it will impact the conversion rate. You can view XDR exchange rates [here](https://www.imf.org/external/np/fin/data/rms_sdrv.aspx).
+The chart below details the cost of compute, storage transmissions and canister calls. A thorough example of how the cost of running a canister on a 13-node subnet is computed can be found [on the wiki](https://wiki.internetcomputer.org/wiki/Comparing_Canister_Cycles_vs_Performance_Counter).
```

Comment:
- is this a `chart`? isn't it rather a `table`?
- avoid the here-link-pattern

Reply:
Revised 'chart' to 'table'